autoload/neomake/utils.vim

```diff
@@ -409,3 +409,22 @@ endfunction
 function! neomake#utils#path_sep() abort
     return neomake#utils#IsRunningWindows() ? ';' : ':'
 endfunction
+
+" Find a file by going up the directories form the start directory
+" and performing glob search for the file.
+function! neomake#utils#FindGlobFile(startDir, file)
+    let currDir = a:startDir
+    let fileFound = ""
+
+    while empty(fileFound)
+        let fileFound = globpath(currDir , a:file, 1)
+        let lastFolder = currDir
+        let currDir = fnamemodify(currDir, ':h')
+
+        if currDir ==# lastFolder
+            break
+        endif
+    endwhile
+
+    return fileFound
+endfunction
```

Comment:
It's fine when it's based on it.. I think Syntastic covers a lot of corner cases already, and we are good to pick that / build upon it.
Having it in a separate utils function is good already, and allows to improve it from there - so if you think it's fine, I'm OK.